account_statement_import_file/migrations/16.0.1.0.1/post-migration.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 Landoo Sistemas de Informacion SL
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if not version:
+        return
+    journals = env["account.journal"].search(
+        [("bank_statements_source", "=", "file_import")]
+    )
+    for journal in journals:
+        journal.bank_statements_source = "file_import_oca"
```

Comment:
- No need of the version check (it's done by the decorator)
- Do things always on SQL (when possible), for performance and for avoiding triggering business logic that may have side effects.

```suggestion
    openupgrade.logged_query(
        env.cr,
        """
        UPDATE account_journal
        SET bank_statements_source = 'file_import_oca'
        WHERE bank_statements_source = 'file_import'
        """
    )
```

Reply:
Changed